exonum/src/node/consensus.rs

```diff
@@ -770,6 +771,22 @@ impl NodeHandler {
                     ).raw()
                         .clone()
                 }
+                RequestData::TransactionsForBlock(ref block_hash) => {
+                    let txs: Vec<_> = self.state
+                        .incomplete_block(block_hash)
+                        .unwrap()
+                        .unknown_txs()
```

Comment:
What's the guarantee that transactions are still unknown at this point? (I understand that this question pertains to `ProposeState` as well.)